diff --git a/bun-sql/postgres/driver.cjs b/bun-sql/postgres/driver.cjs
--- a/bun-sql/postgres/driver.cjs
+++ b/bun-sql/postgres/driver.cjs
@@ -26,7 +26,7 @@ function construct(client, config = {}) {
 	const db = new BunSQLDatabase(dialect, new require_bun_sql_postgres_session.BunSQLSession(client, dialect, relations, {
 		logger,
 		cache: config.cache
-	}), relations, false, true);
+	}), relations, false, false);
 	db.$client = client;
 	db.$cache = config.cache;
 	if (db.$cache) db.$cache["invalidate"] = config.cache?.onMutate;
diff --git a/bun-sql/postgres/driver.js b/bun-sql/postgres/driver.js
--- a/bun-sql/postgres/driver.js
+++ b/bun-sql/postgres/driver.js
@@ -24,7 +24,7 @@ function construct(client, config = {}) {
 	const db = new BunSQLDatabase(dialect, new BunSQLSession(client, dialect, relations, {
 		logger,
 		cache: config.cache
-	}), relations, false, true);
+	}), relations, false, false);
 	db.$client = client;
 	db.$cache = config.cache;
 	if (db.$cache) db.$cache["invalidate"] = config.cache?.onMutate;
